Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py

```diff
@@ -1,19 +1,9 @@
-from mpi4py import MPI
 from pywarpx import picmi
 import numpy as np
 
-##########################
-# MPI communicator setup
-##########################
+import sys
 
-# split processor 0 into separate communicator from others
-comm_world = MPI.COMM_WORLD
-rank = comm_world.Get_rank()
-if rank == 0:
-    color = 0
-else:
-    color = 1
-new_comm = comm_world.Split(color)
+unique = int(sys.argv[1])
```

Comment:
Let's add the Python arg parser here, which will automatically document & validate the mandatory parameter and allows us to also name it on the command line :)

We can also make this a flag that way, e.g., `--unique`.

Reply:
Will do!